app/src/main/java/com/duckduckgo/app/di/DaoModule.kt

```diff
@@ -97,4 +97,7 @@ object DaoModule {
 
     @Provides
     fun syncRelationsDao(database: AppDatabase) = database.syncRelationsDao()
+
+    @Provides
```

Comment:
Do we need to provide from this module? I would assume we can move it to saved-sites-impl module.

Reply:
it’s removed